workers/tester.sh

```diff
@@ -4,7 +4,7 @@
 
 # This script should always run as if it were being called from
 # the directory it lives in.
-script_directory=`dirname "${BASH_SOURCE[0]}"`
+script_directory=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
```

Comment:
`dirname` returns a relative path so for just ensuring that the script is running in the correct location it is sufficient, but to get a full path to that directory this change is necessary.